src/charm.py

```diff
@@ -147,10 +148,18 @@ def _configure_notary_config_file(self):
             self.container.pull(f"{WORKLOAD_CONFIG_PATH}/config/config.yaml")
             logger.info("Config file already created.")
         except ops.pebble.PathError:
-            config_file = open("src/config/config.yaml").read()
             self.container.make_dir(path=f"{WORKLOAD_CONFIG_PATH}/config", make_parents=True)
             self.container.push(
-                path=f"{WORKLOAD_CONFIG_PATH}/config/config.yaml", source=config_file
+                path=f"{WORKLOAD_CONFIG_PATH}/config/config.yaml",
+                source=yaml.dump(
+                    data={
+                        "key_path": "/etc/notary/config/private_key.pem",
```

Comment:
Let's use our `WORKLOAD_CONFIG_PATH` variable here and in cert path

Reply:
done